tox.ini

```diff
@@ -53,6 +53,12 @@ use-flake8-tabs = true
 max-line-length = 100
 tab-width = 4
 
+per-file-ignores =
+    datastore/util.py:F401
+	datastore/abc.py:F401
+	datastore/typing.py:F401
```

Comment:
Please add a comment that we allow F401, which is “module imported but not used”, because these modules only exist to expose the package's public API.